documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -2613,6 +2613,10 @@ endif::community[]
 `false` if not.
 |Indicates whether field names are sanitized to adhere to {link-prefix}:{link-avro-serialization}#avro-naming[Avro naming requirements].
 
+|[[postgresql-property-skipped-operations]]<<postgresql-property-skipped-operations, `skipped.operations`>>
+|
+|Comma-separated list of oplog operations to skip during streaming. Values that you can specify are: `c` for inserts, `u` for updates, `d` for deletes. By default, no operations are skipped.
```

Comment:
```suggestion
|Comma-separated list of oplog operations to skip during streaming. Values that you can specify are: `c` for inserts/create, `u` for updates, `d` for deletes. By default, no operations are skipped.
```

Reply:
I introduced this feature on **MongoDB** first and I extended this feature to **MySQL** too in past. I tried to sync the documentation of those connectors with my new PR. so I did copy and pates from the MongoDB connector Documentation! So if it needs any change It would be changed for all connectors.
In addition, I forget to change `oplog` to Postgres replication log that I'm going to do it.

Reply:
I was about to change that description in a separate PR soon. Would be great if you could apply that change as people (even within the DBZ team) confuse `i` and `c` for that property. Maybe adding `/create` makes it easier to memorize and relate. WDYT?

Reply:
I agree. Should I modify it (I mean also for my past commits) in this PR or should I create the new one?

Reply:
As this is only modifying docs in the light of making it more memorize-able/relatable I would say you can do this in this PR. (I hope the rest of my team will not punish me for that! LOL)

Reply:
So I'm going to do that